RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -1219,12 +1224,12 @@ boolean L12_sonofaPrefix()
 				{
 					set_property("auto_doCombatCopy", "yes");
 				}
-				if (auto_voteMonster())
+				if (auto_voteMonster() && !auto_voteMonster(true))
 				{
 					auto_voteMonster(false, $location[Sonofa Beach], "");
 					return true;
 				}
-				else if (auto_sausageGoblin())
+				else if (auto_sausageGoblin() && !auto_haveVotingBooth())
```

Comment:
These changes mean we only use non-Free Voters to replace as killing LFMs costs an adventure so turning a free fight into a non-free fight is inefficient.
Also we'll only use sausage goblins to replace if voters aren't available.